src/main/java/sparksoniq/exceptions/UnknownFunctionCallException.java

```diff
@@ -39,4 +39,8 @@ public UnknownFunctionCallException(String fnName, int arity, IteratorMetadata i
             iteratorMetadata.getExpressionMetadata()
         );
     }
+
+    public UnknownFunctionCallException(String message, IteratorMetadata iteratorMetadata) {
```

Comment:
Where is this used? I cannot see any new call in the diff.

Reply:
You are right, I probably added that when I added the exception for non-existing function, but then I decided to stick to the existing one. 
Thank you for spotting